Library/Formula/bazel.rb

```diff
@@ -0,0 +1,59 @@
+#
+# Brew Formula for Google's Bazel
+# written by Felix Klein <me@felixklein.at>
+#
+
+class Bazel < Formula
+  homepage "http://bazel.io/"
+
+  #
+  # Version: Bazel 0.0.2 (latest tag from GitHub)
+  # This version is quite old, however there are no newer tags available on GitHub.
+  #
+  url "https://github.com/google/bazel/archive/0.0.2.tar.gz"
+  sha256 "5865356e74622c248248174acd09f38c07f3983867d7e687fa5932f2216e84c0"
+  version "0.0.2"
+
+  #
+  # Use userpaths, otherwise the build script doesn't find the libarchive library
+  #
+  env :userpaths
```

Comment:
Not necessary. Do an ` inreplace ` instead:

```ruby
    inreplace "compile.sh" do |s|
      s.gsub! "$(brew --prefix 2>/dev/null)/Cellar/libarchive/*/include/archive.h 2>/dev/null | head -n1",
              "#{Formula["libarchive"].opt_include}/archive.h | head -n1"
    end
```

The ` head ` would need fixing as well, so you'd have to scope the inreplace per version.